DRAFT: privacy legislation (#44876)
mdn/content: files/en-us/web/privacy/guides/legislation/index.md

---

> title: International privacy laws

Reviewer: Maybe "privacy regulations" might be a better name? Among other things I'm not sure what "International" means here - at least the CCPA isn't international (in the sense that say the International Criminal Court is).

slug: Web/Privacy/Guides/Legislation
page-type: guide
sidebar: privacy
---

> 

Reviewer: This would really benefit from an overview saying something like:

> Respecting a user's privacy is not just a good practice and a way to build trust, it's legally required in various jurisdictions. In this section of our privacy documentation we'll describe two of the most important regulations, the GDPR, which applies to users in the European Union, and the CCPA, which applies to users in California. In each case we'll explain the conditions in which a website is required to comply with the regulation and what compliance looks like.

> - [General Data Protection Regulation (GDPR)](/en-US/docs/Web/Privacy/Guides/Legislation/GDPR)

Reviewer: For the sake of having a sensible ToC, I think this and the CCPA item should have H2 headings. That would also mean you'd need to rejig the following paragraph "to comply with both", maybe moving it up into the preamble.

>   - : The European Union's comprehensive data protection law enacted to enhance individual control over their own data and to unify data protection regulations across EU member states. GDPR governs how organizations handle personal data of EU residents, enhancing individuals' control over their personal information.

Reviewer: ```suggestion
  - : The European Union's comprehensive data protection law gives individuals control over their own data and  unifies data protection regulations across EU member states. GDPR governs how organizations handle the personal data of EU residents.
```

(the first sentence didn't have a verb, and "enhancing individuals' control over their personal information" seemed to dupe "enhance individual control over their own data")

>   - : GDPR is a strict, opt‑in, rights‑heavy regulation that applies to anyone processing EU data. GDPR requires a lawful basis for processing. Consent must be freely given, specific, informed, and unambiguous. Users must actively opt in before data collection unless another lawful basis applies (contract, legal obligation, legitimate interest, etc.).

Reviewer: 1) interestingly, this markup is technically not allowed, see https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#definition_lists :

> Definition lists written using this syntax must consist of pairs of <dt>/<dd> elements. Using this syntax, it's not possible to write a list with more than one consecutive <dt> element or more than one consecutive <dd> element: the parser will treat this as an error. We expect almost all definition lists on MDN will work with this limitation, and for those that do not, authors can fall back to raw HTML.

it does seem to work! But I'm uncomfortable with it all the same.

2) I'm not sure how useful this para is and wonder if we should just keep this detail about _what the regulation says_ in its own page. For instance, what does "strict" really mean, or "rights-heavy", or for that matter "requires a lawful basis for processing"? I also read "opt-in" as meaning "users have to opt into it" which they don't, so that was confusing.

Author: I changed it to h2 and paragraphs, so no longer an issue on this page.
the css docs definitely have DLs with consecutive DDs. A term can have multiple definitions, so it should be allowed. 🤷

- [California Consumer Privacy Act (CCPA)](/en-US/docs/Web/Privacy/Guides/Legislation/CCPA)

>   - : California's privacy law providing residents rights over their personal information and creates requirements on businesses that collect, use, or share that data, including rights to know, delete, opt out, correct, and limit the use of sensitive data.

Reviewer: ```suggestion
  - : California's privacy law giving Californian residents rights over their personal information and creating requirements on businesses that collect, use, or share that data, including rights to know, delete, opt out, correct, and limit the use of sensitive data.
```

>   - : CCPA is business‑threshold‑based, opt‑out, category‑driven legislation that is focused on the sale and sharing of personal data. CCPA allows businesses to collect and use personal information by default but requires data subjects have the right to opt out of sale or sharing of personal information and the right to limit use of sensitive personal information, while requiring Global Privacy Control (GPC) signals be honored.

Reviewer: Similar comments here. I don't know what "business‑threshold‑based" or "category‑driven" mean.

Author: CCPA applies to businesses meeting specific size‑related thresholds, and based on the type of data being collected. CCPA groups info into defined categories, including “sensitive personal information," with different rules apply depending on the data category. This is explained in the CCPA page. Not sure how deep we want to go here just linking to it.

> To comply with both, developers must ensure that systems collect less data, store it transparently, make it retrievable and deletable, and provide user‑controlled privacy choices. [Design and develop for privacy](/en-US/docs/Web/Privacy/Guides/Designing_for_privacy), and have comprehensive privacy policies, security and breach workflows, Data Subject Access Request (DSAR) / consumer request pipelines, complete data inventory with documentation, and a consent / opt-out capabilities.

Reviewer: This is a pretty dense paragraph and I have a few issues with it.

"To comply with both, developers must ensure that systems collect less data" - this can't be true, surely? Less data than what?

"store it transparently" - I don't know what this means.

I think there is also some duplication, e.g.:

- "make it retrievable" is presumably the same essentially as "Data Subject Access Request (DSAR) / consumer request pipelines". If we want to use technical language like DSAR we should link to a definition.
- "user‑controlled privacy choices" surely overlaps with "consent / opt-out capabilities"
- "Design and develop for privacy" is a link to a whole nother article with lots of things that duplicate this list.

## Glossary of terms

> The various privacy laws slightly different vocabulary in defining users, captured data, actions on that data, data processing, and who processes the data. In our documentation, we use GDPR terminology, including the following terms:

Reviewer: I'm uncertain whether unifying the terminology like this is a good idea. Are the concepts defined by the terms the same in both sets of regulations? Even if they are it might be more helpful to list both sets of terms, and in the doc for each regulation use that regulation's version. Otherwise someone reading the regulation needs to translate from our guide.

> - Personal data

Reviewer: Maybe for readability this would be better with the terms bolded. Also if these are GDPR terms, we should link to the GDPR definition. I guess we are really looking at https://gdpr-info.eu/art-4-gdpr/ ?

>   - : Captured data; any information that relates to an individual, generally a user, who can be directly or indirectly identified. This includes names, email addresses, location information, ethnicity, gender, biometric data, religious beliefs, web [cookies](/en-US/docs/Web/HTTP/Guides/Cookies), and even political opinions. Personal data also includes pseudo-anonymous data if it is relatively easy to identify someone from the nominally anonymized data.

Reviewer: Not sure what "pseudo-anonymous data" means.

Author: "nominally anonymized data"

Author: anonymized, but you can figure out who it is easily, not not really anonymized, so "pseudo" anonymized.

- Data processing
  - : Data actions; any actions performed on data, whether automated or manually, including but not limited to collecting, recording, organizing, structuring, storing, using, and deleting data.

- Data subject

>   - : Your user; any person, generally the application user or site visitor, whose data is processed.

Reviewer: I think use of second person is a bit unclear here, maybe we could just say:

```suggestion
  - : The user whose data is being collected and processed.
```

...or something

- Data controller

>   - : You; the person in your organization who handles collectd data and decides why and how personal data is processed.

Reviewer: ```suggestion
    - : The person or organization who determines why and how personal data is processed: that is, the entity who is ultimately responsible for safeguarding the user's data.
```

maybe? It's not just a person as I understand it, it could be the organization.

- Data processor

>   - : Third parties; any third party that processes personal data on your behalf, such as cloud servers, analytics, and email service providers.

Reviewer: ```suggestion
  - : Any third party that processes personal data on the controller's behalf, such as cloud servers, analytics, and email service providers.
```
